docker/Makefile

```diff
@@ -80,6 +80,27 @@ deploy-dev:
 	$(dc) stop app && $(dc) up -d app
 
 
+deploy-stage:
+	# - build section
```

Comment:
Let's separate build and deploy receipt

Reply:
Please, create todo to build the image in CI on merge into the master branch

Reply:
@artemiy312 , stage is required to show how random branch works. So, we can't build correct stage on merging to master